components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -69,10 +71,17 @@ class ToolbarPresenter(
         toolbar.setSearchTerms(session?.searchTerms ?: "")
         toolbar.displayProgress(session?.progress ?: 0)
         updateToolbarSecurity(session?.securityInfo ?: Session.SecurityInfo())
+
+        if (session != null) {
+            updateToolbarTrackingProtection(session)
+        } else {
+            toolbar.siteTrackingProtection = SiteTrackingProtection.OFF_GLOBALLY
+        }
     }
 
     override fun onUrlChanged(session: Session, url: String) {
         renderer.post(url)
+        updateToolbarTrackingProtection(session)
```

Comment:
We spoke offline about this. I think the name may have been confusing.

Reply:
Updated!